.github/workflows/publish-technical-documentation.yml

```diff
@@ -1,36 +1,21 @@
-name: "publish-technical-documentation"
+name: publish-technical-documentation
 
 on:
   push:
-    branches: [main]
-    paths: ["docs/sources/**"]
+    branches:
+      - main
+    paths:
+      - "docs/sources/**"
   workflow_dispatch:
 jobs:
   sync:
     if: github.repository == 'grafana/k6-docs'
+    permissions:
+      contents: read
+      id-token: write
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v4
-
-      - name: Clone website-sync Action
-        # WEBSITE_SYNC_TOKEN is a fine-grained GitHub Personal Access Token that expires.
-        # It must be regenerated in the grafanabot GitHub account and requires a Grafana organization
-        # GitHub administrator to update the organization secret.
-        # The IT helpdesk can update the organization secret.
-        run: git clone --single-branch --no-tags --depth 1 -b master https://grafanabot:${{ secrets.WEBSITE_SYNC_TOKEN }}@github.com/grafana/website-sync ./.github/actions/website-sync
-
-      - name: Publish to website repository (next)
-        uses: ./.github/actions/website-sync
-        id: publish-next
+      - uses: actions/checkout@v4
+      - uses: grafana/writers-toolkit/publish-technical-documentation@publish-technical-documentation/v1
```

Comment:
@jdbaldry is there any chance this would run into an error because of [line 12](https://github.com/grafana/writers-toolkit/blob/main/.github/workflows/publish-technical-documentation.yml#L12) of the workflow?

`if: github.repository == 'grafana/writers-toolkit'`

Reply:
Or is this referring to this file? https://github.com/grafana/writers-toolkit/blob/main/publish-technical-documentation/action.yml

Reply:
Yeah, it's the latter, it refers to the action defined in the file that you linked

Reply:
Thanks for asking! It's always better to clarify rather than assume I did things right. I make a lot of mistakes :)

Reply:
❤️